### ISSUE TITLE
Update user guide to be suitable for new `view` command

### ISSUE THREAD

**Issue by yuntongzhang** [CS2103-AY1819S1-W14-3/main#123]
- [x] Remove the outdated documentation for view commands.
- [x] Add new available options for the new `view` command.

This should be done before the mock testing on **2nd Nov**.

**Comment by yuntongzhang** [comment]
Comment here if you are done with your part, so that we can close this issue.
@omegafishy @jeffypie369 @GAOJIAXIN666 @snajef

**Comment by omegafishy** [comment]
I just checked the latest generated gh pages on the DG issues above, and the changes have been effected.

**Comment by omegafishy** [comment]
@yuntongzhang please reference the relevant issue #200 in future, I missed out the extra details that were not included in this issue. It appears #200 is a more updated duplicate of this issue.

**Comment by yuntongzhang** [comment]
#200 was created yesterday and assigned to @snajef 
Now we can close this issue yay
